website/templates/project/project_header.mako

```diff
@@ -68,8 +81,18 @@
                         % if user['has_read_permissions'] and not node['is_registration'] or (node['is_registration'] and 'admin' in user['permissions']):
                             <li><a href="${node['url']}settings/">Settings</a></li>
                         % endif
-
                     % endif
+                    % if user['can_comment'] or node['has_comments']:
```

Comment:
It refers to the "Comments" link in the project nav bar on small screen sizes:  
![screen shot 2016-01-04 at 3 25 22 pm](https://cloud.githubusercontent.com/assets/6414394/12099624/a73421bc-b2f7-11e5-934f-afdaa96f7f56.png)